wd_helper.au3

```diff
@@ -1,7 +1,11 @@
 #include-once
+; standard UDF's
 #include <File.au3> ; Needed For _WD_UpdateDriver
 #include <InetConstants.au3>
+
+; WebDriver related UDF's
 #include "wd_core.au3"
+#include "wd_capabilities.au3"
```

Comment:
I would expect `wd_capabilities.au3` to be included here.

Reply:
Corrected above comment to reflect that wd_capabilities.au3 should *not* be included in wd_helper. wd_core.au3 is needed because it contains functions called by the "helper" functtions, but this isn't true about wd_capabilities.au3

Reply:
So user of WD***.au3 should intentionally include helper and capabilities as a separate files ?

Reply:
Yes. wd_core.au3 should function independent of the other "add on" modules. If the user needs more functionality, then they have the option to #include one or more files.